pulp_container/app/viewsets.py

```diff
@@ -1082,8 +1108,19 @@ class ContainerPushRepositoryVersionViewSet(
                 "has_distribution_perms:container.view_containerdistribution",
             },
         ],
+        "queryset_scoping": {
+            "function": "get_push_repos_qs",
+            "parameters": {
+                "ns_perm": "container.view_containernamespace",
+                "dist_perm": "container.view_containerdistribution",
+            },
+        },
     }
 
+    def get_push_repos_qs(self, qs, ns_perm, dist_perm):
```

Comment:
@gerrod3 for the viewsets that have a parent viewset, does it make sense to just add a check here in core and just take the queryset_scoping automatically from the parent viewset https://github.com/pulp/pulpcore/blob/main/pulpcore/app/access_policy.py#L68 or you see a usecase where 'child viewset' would have it's own scoping?

Reply:
nevermind, i realized that queryset_scoping it is not even needed here.